tests/phpunit/tests/media.php

```diff
@@ -4318,13 +4339,19 @@ public function test_wp_get_loading_optimization_attributes( $context ) {
 
```

Comment:
There are three assertions above here that I think you missed to update, which is why this test is failing. In all of those assertions, `decoding="async"` must also be present.